src/pages/index.tsx

```diff
@@ -352,6 +352,10 @@ export default function Home() {
               setButtonStatus(!buttonStatus);
             };
 
+            useEffect(() => {
+              setCardImage("")
+            },[content.image[i]])
+
```

Comment:
This may lead to issues. useEffect is not meant to be used inside a .map, the depedency array is also incorrect.

Reply:
Also it seems liek you're trying to useState within the map, this is a clear indication that this component needs to be extracted into a seperate component. Generally you never want to see useState used inside a .map directly, that includes creating a function like handleButtonStatus.